src/freenas/usr/local/bin/nvmf-connect.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+RECONNECT_TIMEOUT_SEC=5
+discovery_dev=$(echo "$@" | grep -oP '(?<=--device=)\w+')
+discovery_addr=$(</sys/class/nvme/${discovery_dev}/address)
+queue="/var/tmp/${discovery_dev}-fabric-queue"
```

Comment:
Maybe use a path under `/var/run`. `/var/run/middleware` might be an option. Sometimes files can get cleaned out of /tmp at unfortunate times.

Reply:
@anodos325 - I can change it as you suggested but `/var/tmp` is not mounted on `tmpfs` like `/tmp`.